bundles/org.openhab.binding.fmiweather/src/main/feature/feature.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="org.openhab.binding.fmiweather-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
+    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${project.version}/xml/features</repository>
```

Comment:
This should fix your dependency problem.
```suggestion
    <repository>mvn:org.openhab.core.features.karaf/org.openhab.core.features.karaf.openhab-core/${ohc.version}/xml/features</repository>
```

Reply:
That works for my setup. Thanks @cpumeister for the debugging

Reply:
Much appreciated!